src/main/java/emu/grasscutter/game/gacha/GachaBanner.java

```diff
@@ -96,7 +126,7 @@ public GachaInfo toProto() {
 		return toProto("");
 	}
 	public GachaInfo toProto(String sessionKey) {
-		String record = "http" + (Grasscutter.getConfig().getDispatchOptions().FrontHTTPS ? "s" : "") + "://"
+		String record = "https://"
```

Comment:
Please do not revert made in other PRs changed arbitrarily. Ensure your fork is up-to-date or be very careful when merging conflicts.

Reply:
https://github.com/Grasscutters/Grasscutter/pull/649